apps/mull-ui/src/app/components/event-card/event-card.spec.tsx

```diff
@@ -1,22 +1,46 @@
+import { MockedProvider } from '@apollo/client/testing';
 import { render } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
 import React from 'react';
+import { Router } from 'react-router-dom';
 import renderer from 'react-test-renderer';
 import { dummyEvent } from '../../../constants';
 import EventCard from './event-card';
 
 describe('EventCard', () => {
+  const history = createMemoryHistory();
```

Comment:
Careful when declaring stuff outside of tests like this, since tests might start affecting each other.